src/Uno.XamlMerge.Tests/Scenarios/When_Different_Namespace_Should_Be_Merged/Input_Dictionary_1.xml

```diff
@@ -1,8 +1,12 @@
 ﻿<ResourceDictionary
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-	xmlns:wasm="http://uno.ui/wasm#using:NS1">
+	xmlns:wasm="http://uno.ui/wasm#using:NS1"
+  xmlns:android="http://uno.ui/android#using:AndroidNS"
+  xmlns:ios="http://uno.ui/ios">
```

Comment:
Let's keep existing tests as they are still valid, and add new ones, it will make it easier to validate individual regressions.